src/main/java/core/basesyntax/resourses/database.csv

```diff
@@ -0,0 +1,9 @@
+type,fruit,quantity
+b,banana,20
+b,apple,100
```

Comment:
why do we have files here? 
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/55d715e1-7904-4526-a3ee-ad44201ed8bf)